Formula/dotnet.rb

```diff
@@ -68,19 +52,16 @@ def install
       ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
     end
 
-    # TODO: Remove this in future release with .NET runtime v6.0.2+
-    (buildpath/"src/SourceBuild/tarball/patches/runtime").install resource("runtime-clang13-patch")
-
     Dir.mktmpdir do |sourcedir|
       system "./build.sh", "/p:ArcadeBuildTarball=true", "/p:TarballDir=#{sourcedir}"
 
       cd sourcedir
-      # Disable package validation in source-build for reliability
-      # PR ref: https://github.com/dotnet/runtime/pull/60881
-      # TODO: Remove this in the next release
-      inreplace Dir["src/runtime.*/eng/packaging.targets"].first,
-                "<EnablePackageValidation>true</EnablePackageValidation>",
-                "<EnablePackageValidation>false</EnablePackageValidation>"
+      # Workaround for error MSB4018 while building 'installer in tarball' due
+      # to trying to find aspnetcore-runtime-internal v6.0.0 rather than current.
+      # TODO: Remove when packaging is fixed
+      inreplace Dir["src/installer.*/src/redist/targets/GenerateLayout.targets"].first,
+                "$(MicrosoftAspNetCoreAppRuntimePackageVersion)",
+                "$(MicrosoftAspNetCoreAppRuntimewinx64PackageVersion)"
```

Comment:
Before merge, main thing to check upstream is why `MicrosoftAspNetCoreAppRuntimePackageVersion` is using **6.0.0** and failing.

`MicrosoftAspNetCoreAppRuntimewinx64PackageVersion` is set to **6.0.2** and it looks like build is supposed to use `MicrosoftAspNetCoreAppRuntimePackageVersion = MicrosoftAspNetCoreAppRuntimewinx64PackageVersion`